core/src/config.jl

```diff
@@ -49,6 +49,15 @@ end
 
 snake_case(sym::Symbol)::Symbol = Symbol(snake_case(String(sym)))
 
+"Convert a string from snake_case to CamelCase."
+function camel_case(snake_case::AbstractString)::String
```

Comment:
Take a look at titlecase

Reply:
![image](https://github.com/user-attachments/assets/f8b8c2de-11f8-47a0-bd16-005a1a1faf40)

Reply:
If it should become CamelCase, I would probably go for: `titlecase()`, and then `replace('_' => '')`

Reply:
I want this function to return the same string when the input is already camel case. 
![image](https://github.com/user-attachments/assets/8d6d6ebb-9e13-47ba-9235-38eb00fb588d)

Seems titlecase() is not able to do it because it treats CamelCase as one word and return Camelcase.

The old way is the same amount of lines but more robust